### DISCUSSION TITLE
第二次的活动大家觉得怎么样？

### DISCUSSION THREAD

**Discussion by suredream** [suredream/hack-your-ds-interview#18] [General]
你对于活动的内容 [0-没有学到，1-一般，2-很好]
活动的长度 [0-太短，1-太长，2-合理]
话题的深度 [0-太浅，1-太深，2-合理]
活动的节奏 [0-太慢，1-太快，2-合理]
参与度 [0-难以参与，1-想参与但无方式，2-暂无感觉]

**Comment by JJ-Zhang-DS** [comment]
你对于活动的内容 [2-很好]
活动的长度 [2-合理]
话题的深度 [2-合理]
活动的节奏 [2-合理]
参与度 [1-想参与但无方式]

能听到很多DS业内人士的切身体验，感觉受益匪浅。

**Reply by suredream** [reply]
您希望能以什么方式更好地参与其中呢？ @zjj1128

**Reply by JJ-Zhang-DS** [reply]
我还是beginner，感觉目前还难以作为主讲人参与。目前的参与更多的是学习。去review（学习）别的notebook可能更现实一些。
我想以后做project时候，可以把自己的project拿来讲讲，或者找工作的时候用take home challenge拿来讲？
也许您或者其他人有idea可以增加初学者的参与度，可以讨论一下。

**Reply by suredream** [reply]
您愿意更多参与 review 和 wiki 吗？比如一次 review 结束后，由你来整理笔记和 wiki，有问题的话与 host 协调或自己展开一些 google，最终产出是比较规范的文档？host 和其它人可以再次 review 你的文档，防止出错。 @zjj1128

**Reply by JJ-Zhang-DS** [reply]
可以，如果您觉得我能胜任的话。
也许下次我可试试，看看有没有能力完成。

**Reply by suredream** [reply]
那太好了，下次的活动是 LSTM 模型，希望有兴趣

**Reply by JJ-Zhang-DS** [reply]
好。谢谢信任。
